src/index/IndexImpl.cpp

```diff
@@ -70,18 +72,7 @@ IndexBuilderDataAsFirstPermutationSorter IndexImpl::createIdTriplesAndVocab(
 // _____________________________________________________________________________
 std::unique_ptr<RdfParserBase> IndexImpl::makeRdfParser(
     const std::vector<Index::InputFileSpecification>& files) const {
-  auto makeRdfParserImpl =
-      [this, &files]<int useCtre>() -> std::unique_ptr<RdfParserBase> {
-    using TokenizerT =
-        std::conditional_t<useCtre == 1, TokenizerCtre, Tokenizer>;
-    return std::make_unique<RdfMultifileParser<TokenizerT>>(
-        files, this->parserBufferSize());
-  };
-
-  // `callFixedSize` litfts runtime integers to compile time integers. We use it
-  // here to create the correct combinations of template arguments.
-  return ad_utility::callFixedSize(std::array{onlyAsciiTurtlePrefixes_ ? 1 : 0},
-                                   makeRdfParserImpl);
+  return std::make_unique<RdfMultifileParser>(files, parserBufferSize());
```

Comment:
Does this still correctly handle the information about the used tokenizers
(ie. is that part of the `InputFileSpecification` and hence the template argument of the `RdfMultifileParser` was already redundant before you removed it?

Reply:
I checked, the template argument was completely unused before I removed it.